src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -120,6 +120,8 @@ public SwitchParameter NoClobber
 
```

Comment:
@IISResetMe, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @(@{InputObject=ConvertFrom-Clixml; SideIndicator=<=}, @{InputObject=ConvertTo-Clixml; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 540
540:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Write-Verbose; ConfirmImpact=None; SideIndicator=<=}, @{Name=Write-Warning; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 564
564:         $result | Should -BeNullOrEmpty
```
CliXml test.Import-CliXML.test follow-up for WinBlue: 161470 - Export-CliXml errors in WhatIf scenarios
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\clixml.tests.ps1: line 169
169:             Export-Clixml -Path $testPath -InputObject "string" -WhatIf

```
XmlCommand DRT basic functionality Tests.Import-Clixml -IncludeTotalCount always returns unknown total count
```powershell
Expected strings to be the same, but they were different.
Expected length: 19
Actual length:   0
Strings differ at index 0.
Expected: 'Unknown total count'
But was:  ''
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\XMLCommand.Tests.ps1: line 120
120: 		$out[0].ToString() | Should -BeExactly "Unknown total count"
```
XmlCommand DRT basic functionality Tests.Import-Clixml -First and -Skip work together for simple types
```powershell
Expected 2, but got 4.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\XMLCommand.Tests.ps1: line 126
126: 		$out.Count | Should -Be 2
```